runtime/doc/lua.txt

```diff
@@ -3670,4 +3670,63 @@ totable({f}, {...})                                       *vim.iter.totable()*
     Return: ~
         (table)
 
+
+==============================================================================
+Lua module: vim.snippet                                          *vim.snippet*
+
+active()                                                    *snippet.active()*
+    Returns `true` if there's an active snippet in the current buffer.
+
+    Return: ~
+        (boolean)
+
+exit()                                                        *snippet.exit()*
+    Exits the current snippet.
+
+expand({input})                                             *snippet.expand()*
+    Expands the given snippet text. Refer to https://microsoft.github.io/language-server-protocol/specification/#snippet_syntax for the specification of valid input.
+
+    Tabstops are highlighted with hl-SnippetTabstop.
+
+    Parameters: ~
+      • {input}  (string)
+
+jump({direction})                                             *snippet.jump()*
+    Jumps within the active snippet in the given direction. If the jump isn't
+    possible, the function call does nothing.
+
+    You can use this function to navigate a snippet as follows: >lua
+        vim.keymap.set({ 'i', 's' }, '<Tab>', function()
+           if vim.snippet.jumpable(1) then
+             return '<cmd>lua vim.snippet.jump(1)<cr>'
+           else
+             return '<Tab>'
+           end
+         end, { expr = true })
+<
+
+    Parameters: ~
+      • {direction}  (vim.snippet.Direction) Navigation direction. -1 for
+                     previous, 1 for next.
+
+jumpable({direction})                                     *snippet.jumpable()*
+    Returns `true` if there is an active snippet which can be jumped in the
```

Comment:
Seems redundant with `active()`. If we [replace](https://github.com/neovim/neovim/pull/25301/files#r1392948224) `active` with `get(): {id: number, jumpable: boolean|nil}` then we could eliminate this.

Reply:
Not redundant. Say the cursor is modifying the placeholder of the last tabstop. Then `vim.snippet.active()` returns true but `vim.snippet.jumpable(1)` does not.

Reply:
Sorry, I meant it would be redundant if we have `get()`